### ISSUE TITLE
Fix Getting started instruction

### ISSUE THREAD

**Issue by mihbort** [innosoft-pro/label-them#22]
Hi guys! I've just tried your new release from branch develop and have some concerns:

- In [getting started](https://github.com/innosoft-pro/label-them/wiki/Getting-started-YandexToloka) is better to mention that there is possible took release version and not to build yourself.
- During the building there is two missing steps in instruction: install `grunt-contrib-concat` and `grunt-contrib-cssmin`. Without them isn't working. 
- Very hard to understand what is necessary to generate the `.tsv` file, how to use the python script. It isn't mentioned that files has to be placed locally on your computer, not in Yandex.disk. Also not mentioned how to get proper `.json` file.

**Comment by AlNedorezov** [comment]
Hi, @mihbort. Thank you for your feedback!

Note that the latest version of the system can be found in the [develop-toloka branch](https://github.com/innosoft-pro/label-them/tree/develop-toloka), and not in the "develop" branch, which is mentioned in the [README file](https://github.com/innosoft-pro/label-them/blob/master/README.md).

> In getting started is better to mention that there is possible took release version and not to build yourself.

It is actually mentioned in the note after step 7 on the [getting started wiki page](https://github.com/innosoft-pro/label-them/wiki/Getting-started-YandexToloka).

> During the building there is two missing steps in instruction: install grunt-contrib-concat and grunt-contrib-cssmin. Without them isn't working.

Thank you, once again. We will check out if `npm install grunt-contrib-concat` should be added to the instruction.
As for the `grunt-contrib-cssmin`, it is mentioned as a prerequisite both in the [README file](https://github.com/innosoft-pro/label-them/blob/master/README.md), and on the [Generating source files for YandexToloka wiki page](https://github.com/innosoft-pro/label-them/wiki/Generating-source-files-for-YandexToloka).

> Very hard to understand what is necessary to generate the .tsv file, how to use the python script. It isn't mentioned that files has to be placed locally on your computer, not in Yandex.disk. Also not mentioned how to get proper .json file.

We'll take that into account and will try to reduce possible ambiguities. Although, it is mentioned in the [Generating a TSV instruction](https://github.com/innosoft-pro/label-them/wiki/Getting-started-YandexToloka#generating-a-tsv) that path to files is a "relative path to the folder that you uploaded to Yandex.Toloka", which implies that it is stored on your machine.

As for the json, it's structure is described on the [Classes and parameters JSON description wiki page](https://github.com/innosoft-pro/label-them/wiki/Classes-and-parameters-JSON-description).

**Comment by mihbort** [comment]
> It is actually mentioned in the note after step 7 on the getting started wiki page.

Please, swap these two Notes. It is likely that person would start doing the first one and only after that reads the second.

> "relative path to the folder that you uploaded to Yandex.Toloka"

This part can be formulated the following way: "path to folder on your local machine that you uploaded to Yandex.Toloka. It is needed to specify all it's content in .tsv file"
It may be worth to split generating in more steps.

> As for the json, it's structure is described on the Classes and parameters JSON description wiki page.

Add link to this document from Generating instruction.

**Comment by ruslanmustafin** [comment]
Hi, thanks for the feedback!

>It isn't mentioned that files has to be placed locally on your computer, not in Yandex.disk 

In one of the previous steps in the tutorial, you were asked to upload local data files to a specific folder on Yandex.Disk. This implies having a local copy of the files on a computer that you use to generate .tsv, but I agree that it might not be straightforward so we will add it to the docs. 

> This part can be formulated the following way: "path to folder on your local machine that you uploaded to Yandex.Toloka. It is needed to specify all it's content in .tsv file"

I agree, this is a better way to formulate this.

**Comment by AlNedorezov** [comment]
@mihbort, thank you for your suggestions!

> Please, swap these two Notes. It is likely that person would start doing the first one and only after that reads the second.

I have swapped and reformulated the notes a little. At the moment, the "easier" way to get source files for Yandex.Toloka, from releases page, goes first.

> Add link to this document from Generating instruction.

I have added a link to the "Classes and parameters JSON description" wiki page to the [Generating a TSV guide](https://github.com/innosoft-pro/label-them/wiki/Getting-started-YandexToloka#generating-a-tsv).

**Comment by mihbort** [comment]
@AlNedorezov @ruslanmustafin 
> I agree that it might not be straightforward so we will add it to the docs.

Just add it and I close the issue, the rest of the file is good.

**Comment by AlNedorezov** [comment]
@mihbort 

> Just add it and I close the issue, the rest of the file is good.

I have noted in the [Generating a TSV guide](https://github.com/innosoft-pro/label-them/wiki/Getting-started-YandexToloka#generating-a-tsv), that a path shoud point to the folder on the local machine, and that all of the stated folders content will be specified in a generated `.tsv` file

**Comment by mihbort** [comment]
All right

**Comment by nmingazov** [comment]
Also, what about to delete `develop` branch? It hasn't been updated for 3 months straight and could confuse newcomers.

**Comment by nmingazov** [comment]
Also, talking about branches - why we don't update `master` with the latest release? Version 1.4 was released 3 days ago while latest commit in `master` was 21 days ago.

**Comment by AlNedorezov** [comment]
@nmingazov

> Also, what about to delete develop branch? It hasn't been updated for 3 months straight and could confuse newcomers.

I have deleted `develop` branch, as it, indeed, served no purpose, and may have confused newcomers.

> Also, talking about branches - why we don't update master with the latest release? Version 1.4 was released 3 days ago while latest commit in master was 21 days ago.

We didn't update `master` branch with the new release, since we are still working on reducing the technical debt introduced by the new changes. However, I have made a decision to publish a release which contains delayed acceptance, since this feature was requested ASAP, and it's straightforward functionality was tested.
We plan to update `master` branch in a few days.
